vm/src/memory/call_stack.cpp

```diff
@@ -119,4 +126,47 @@ bool CallStack::is_empty() const {
 void CallStack::clear() {
     frame_count = 0;
     frame_headers_pos = 0;
+}
+
+call_stack_iterator CallStack::begin() {
+    return call_stack_iterator(this, frame_headers_pos, size());
+}
+
+call_stack_iterator CallStack::back() {
+    if (is_empty())
+        return end();
+    return call_stack_iterator(this, 0, 1);
+}
+
```

Comment:
```suggestion
```
`back()` returns a reference to the last element in the stack, not an iterator. Given that `front()` does not exist, it is a strange addition here.